subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenPomResolveIntegrationTest.groovy

```diff
@@ -30,13 +30,14 @@ class MavenPomResolveIntegrationTest extends AbstractHttpDependencyResolutionTes
     <version>1.2</version>
     <distributionManagement>
         <relocation>
```

Comment:
Can you also add an integration tests for the following two test cases:

1. The relocated module cannot be resolved.
2. The relocated module relocates to another location.